test/e2e/bash/README.md

```diff
@@ -0,0 +1,14 @@
+#  Kubestellar end to end testing
+
+**PRE-REQ**: All of these tests use three `kind` clusters, so you need to [raise the relevant OS limits](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files).
+
+This test is an executable variant of the "multi-cluster workload deployment with kubectl" scenario in [the examples doc](../../../docs/content/direct/examples.md). This test has the same prerequisites as the cited one. This test can test either (a) the  local copy of the repo or (b) the release identified in the kubestellar PostCreateHook (which will be the last release created, regardless of quality, except for that brief time when it identifies the release about to be made). Testing the local copy is the default behavior; to test the release identified in the PostCreateHook, pass `--released` on the command line of `run-test.sh`.
+
+## Running the test using a script
+
+Starting from a local directory containing the git repo, do the following.
+
+```
+cd test/e2e/bash
+../run-test.sh
+```
```

Comment:
This is entirely redundant with `test/e2e/README.md`, right?

Reply:
I added that README.md just to simplify the ability to run each test independently (bash & ginkgo).

Reply:
Even without this section, `test/e2e/README.md` should be telling the user how to run each test independently.

Reply:
`test/e2e/README.md` is already providing instruction to the user on how to run each test independently. We can remove the additional readme file at `test/e2e/bash/README.md` to avoid any confusion

Reply:
What is needed in `test/e2e/bash/README.md`, if it is not in `test/e2e/README.md`, is a description of the scenario.

Reply:
Which scenario? could you clarify it? thx!

Reply:
The scenario that is in this directory's use-kubestellar script. The description is the following text from the above paragraph.

> This test is an executable variant of the "multi-cluster workload deployment with kubectl" scenario in [the examples doc](../../../docs/content/direct/examples.md). This test has the same prerequisites as the cited one.

Reply:
Fixed! Added a description for the scenario